### PR TITLE
Fix a bug whereby Unilateral would be forgotten in the ContractModel tests

### PR DESCRIPTION
**Pull request by MaximilianAlgehed**
Pre-submit checklist:
- Branch
    - [x] Tests are provided (if possible)
    - [x] Commit sequence broadly makes sense
    - [x] Key commits have useful messages
    - [x] Relevant tickets are mentioned in commit messages
    - [x] Formatting, materialized Nix files, PNG optimization, etc. are updated
- PR
    - [x] Self-reviewed the diff
    - [x] Useful pull request description
    - [x] Reviewer requested

### CODE REVIEW

**Comment by MaximilianAlgehed** [comment]
@sjoerdvisscher hate to bother you, but this is a quick bugfix that we would appreciate to get into main ASAP.

File: plutus-contract/src/Plutus/Contract/Test/ContractModel/Internal.hs [hunk 1]
```diff
@@ -1579,30 +1580,34 @@ checkNoLockedFundsProofWithWiggleRoomFast wiggle = checkNoLockedFundsProofWithWi
...
-  -> (Actions model -> Property)
+  -> (StateModel.Actions (ModelState model) -> Property)
   -> NoLockedFundsProof model
   -> Property
 checkNoLockedFundsProofWithWiggleRoom' wiggle run NoLockedFundsProof{nlfpMainStrategy   = mainStrat,
                                                                      nlfpWalletStrategy = walletStrat } =
     forAllDL anyActions_ $ \ (Actions as) ->
     forAllUniqueDL (nextVarIdx as) (stateAfter $ Actions as) mainStrat $ \ (Actions as') ->
           let s = stateAfter $ Actions (as ++ as') in
-            foldl (QC..&&.) (run (Actions $ as ++ as') QC..&&. (counterexample "Main strategy" . counterexample (show . Actions $ as ++ as') $ prop s))
```
Comment (sjoerdvisscher) [inline plutus-contract/src/Plutus/Contract/Test/ContractModel/Internal.hs:1590]: Use the new `Semigroup Action` instance here in the next refactor.

**Review by sjoerdvisscher — APPROVED** [review]

File: plutus-contract/src/Plutus/Contract/Test/ContractModel/Internal.hs (hunk 1 above)
Comment (MaximilianAlgehed) [reply plutus-contract/src/Plutus/Contract/Test/ContractModel/Internal.hs:1590]: A different kind of actions ;)

Comment (MaximilianAlgehed) [reply plutus-contract/src/Plutus/Contract/Test/ContractModel/Internal.hs:1590]: But we should add a semigroup instance for those actions too...

**Pull request merged**